share/server/views.js

```diff
@@ -120,10 +120,10 @@ var Views = (function() {
       Couch.recursivelySeal(doc);
 
       var buf = [];
-      for each (fun in State.funs) {
+      for (fun in State.funs) {
```

Comment:
Same question here about the lack of declaration for the `fun` binding.

Reply:
Good call, let me change it to `fun_idx`